libs/types/src/election.ts

```diff
@@ -142,11 +142,6 @@ export const WriteInCandidateSchema: z.ZodSchema<WriteInCandidate> = z.object({
   partyIds: z.array(PartyIdSchema).optional(),
 });
 
-export const writeInCandidate: Candidate = {
-  id: 'write-in',
-  name: 'Write-In',
-  isWriteIn: true,
-};
```

Comment:
While not strictly necessary, I thought it was better to keep this under `tabulation` rather than `election` because it really is a tabulation tool. Also, use variable naming to indicate it represents a generic or pending write-in, rather than perhaps thinking it is an actual write-in.